homeassistant/components/suez_water/sensor.py

```diff
@@ -65,6 +66,7 @@ class SuezSensor(SensorEntity):
     _attr_icon = "mdi:water-pump"
     _attr_native_unit_of_measurement = UnitOfVolume.LITERS
     _attr_device_class = SensorDeviceClass.WATER
+    _attr_state_class: SensorStateClass = SensorStateClass.TOTAL
```

Comment:
Please make a separate PR for changes that aren't related to the library bump.

Reply:
Right